x-pack/plugins/session_view/public/components/process_tree_alert/styles.ts

```diff
@@ -68,15 +68,11 @@ export const useStyles = ({ isInvestigated, isSelected }: StylesDeps) => {
 
     const alertRuleName: CSSObject = {
       ...alertRowItem,
-      maxWidth: '70%',
```

Comment:
synced with @codearos about that and we don't have a need to limit alert content

Reply:
iirc, this is a workaround to make sure the rule name doesn't squeeze other elements in the row. If this was removed, a long rule name can make the alert (danger) icon appear smaller than it should

Reply:
Got it, thanks for catching that, yes by default flex will make larger elements "eat" the spacing of other elements, making them shrink, which would make the alert list elements appear different based on the description. We can prevent that behavior by adding `flex-shrink: 0` to the item we don't want to shrink.

I just pushed an update to add `flex-shrink: 0` to the expand button, it seems it wasn't needed for the warning icon, since EuiIcons have `flex-shrink: 0` already.

It seems to be working fine now:

![image](https://user-images.githubusercontent.com/19270322/165523899-c03f2037-258e-41ce-a472-9f6ad8d44fe5.png)

Items are following the same spacing no matter the length of the description:
![image](https://user-images.githubusercontent.com/19270322/165525532-326caed0-ed27-4294-b2c4-5ed4f230fd36.png)

Reply:
Awesome! Thanks Paulo :)